### PR TITLE
Update outdated files in the dev-1.19-ko.6 branch

### LINKED ISSUE #25284: Outdated files in the upstream/dev-1.19-ko.6 branch
# This is a Bug Report
## Problem
Outdated files in the upstream/dev-1.19-ko.6 branch.

### 42 files to be modified 
1. [ ] content/en/_index.html 1 file changed, 7 insertions(+), 2 deletions(-)
1. [ ] content/en/case-studies/newyorktimes/index.html 1 file changed, 3 insertions(+), 4 deletions(-)
1. [ ] content/en/docs/concepts/architecture/controller.md 1 file changed, 2 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/concepts/architecture/nodes.md 1 file changed, 1 deletion(-)
1. [ ] content/en/docs/concepts/cluster-administration/_index.md 1 file changed, 2 deletions(-)
1. [ ] content/en/docs/concepts/cluster-administration/addons.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/concepts/cluster-administration/networking.md 1 file changed, 6 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/configuration/configmap.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/configuration/pod-priority-preemption.md 1 file changed, 2 insertions(+), 4 deletions(-)
1. [ ] content/en/docs/concepts/policy/resource-quotas.md 1 file changed, 2 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/concepts/scheduling-eviction/assign-pod-node.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/scheduling-eviction/resource-bin-packing.md 1 file changed, 105 insertions(+), 79 deletions(-)
1. [ ] content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md 1 file changed, 7 insertions(+), 7 deletions(-)
1. [ ] content/en/docs/concepts/services-networking/ingress-controllers.md 1 file changed, 3 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/services-networking/network-policies.md 1 file changed, 3 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/concepts/storage/storage-classes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/controllers/statefulset.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/pods/_index.md 1 file changed, 12 insertions(+), 8 deletions(-)
1. [ ] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md 1 file changed, 2 deletions(-)
1. [ ] content/en/docs/contribute/_index.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/contribute/new-content/overview.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/home/_index.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/reference/access-authn-authz/service-accounts-admin.md 1 file changed, 72 insertions(+), 60 deletions(-)
1. [ ] content/en/docs/reference/command-line-tools-reference/feature-gates.md 1 file changed, 5 insertions(+), 5 deletions(-)
1. [ ] content/en/docs/reference/kubectl/cheatsheet.md 1 file changed, 7 insertions(+)
1. [ ] content/en/docs/reference/kubectl/overview.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/setup/best-practices/cluster-large.md 1 file changed, 85 insertions(+), 89 deletions(-)
1. [ ] content/en/docs/setup/best-practices/node-conformance.md 1 file changed, 5 insertions(+), 4 deletions(-)
1. [ ] content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/setup/release/notes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/configure-pod-container/quality-service-pod.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/tasks/debug-application-cluster/debug-pod-replication-controller.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/extend-kubectl/kubectl-plugins.md 1 file changed, 3 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md 1 file changed, 27 insertions(+), 26 deletions(-)
1. [ ] content/en/docs/tasks/run-application/horizontal-pod-autoscale.md 1 file changed, 7 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/tools/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tutorials/kubernetes-basics/_index.html 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/tutorials/stateful-application/zookeeper.md 1 file changed, 4 insertions(+), 4 deletions(-)
1. [ ] content/en/examples/pods/pod-with-pod-affinity.yaml 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/training/_index.html 1 file changed, 22 insertions(+), 15 deletions(-)

### 2 files to be deleted 
1. [ ] content/en/docs/setup/production-environment/turnkey/_index.md
1. [ ] content/en/docs/tasks/administer-cluster/cluster-management.md

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `upstream/dev-1.19-ko.6` branch.

For example:
```
# checkout `upstream/dev-1.19-ko.6`
...
# check what is updated in the upstream 
git diff upstream/dev-1.19-ko.5 upstream/dev-1.19-ko.6 -- content/en/_index.html
# apply changes to content/ko
vi content/ko/_index.html
...
# commit and push
...
# make PR to `upstream/dev-1.19-ko.6`
```

## Pages to Update

/language ko

### PR DESCRIPTION
**Pull request by pjhwa**
Closes #25284 

Update outdated files in the upstream/dev-1.19-ko.6 branch.

41 files are updated
- [x] content/en/_index.html 1 file changed, 7 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/architecture/controller.md 1 file changed, 2 insertions(+), 3 deletions(-)
- [x] content/en/docs/concepts/architecture/nodes.md 1 file changed, 1 deletion(-)
- [x] content/en/docs/concepts/cluster-administration/_index.md 1 file changed, 2 deletions(-)
- [x] content/en/docs/concepts/cluster-administration/addons.md 1 file changed, 1 insertion(+)
- [x] content/en/docs/concepts/cluster-administration/networking.md 1 file changed, 6 insertions(+), 1 deletion(-)
- [x] content/en/docs/concepts/configuration/configmap.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/configuration/pod-priority-preemption.md 1 file changed, 2 insertions(+), 4 deletions(-)
- [x] content/en/docs/concepts/policy/resource-quotas.md 1 file changed, 2 insertions(+), 3 deletions(-)
- [x] content/en/docs/concepts/scheduling-eviction/resource-bin-packing.md 1 file changed, 105 insertions(+), 79 deletions(-)
- [x] content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md 1 file changed, 7 insertions(+), 7 deletions(-)
- [x] content/en/docs/concepts/services-networking/ingress-controllers.md 1 file changed, 3 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/services-networking/network-policies.md 1 file changed, 3 insertions(+), 3 deletions(-)
- [x] content/en/docs/concepts/storage/storage-classes.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/pods/_index.md 1 file changed, 12 insertions(+), 8 deletions(-)
- [x] content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md 1 file changed, 2 deletions(-)
- [x] content/en/docs/contribute/_index.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/home/_index.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/reference/access-authn-authz/service-accounts-admin.md 1 file changed, 72 insertions(+), 60 deletions(-)
- [x] content/en/docs/reference/command-line-tools-reference/feature-gates.md 1 file changed, 5 insertions(+), 5 deletions(-)
- [x] content/en/docs/reference/kubectl/cheatsheet.md 1 file changed, 7 insertions(+)
- [x] content/en/docs/reference/kubectl/overview.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/setup/best-practices/cluster-large.md 1 file changed, 85 insertions(+), 89 deletions(-)
- [x] content/en/docs/setup/best-practices/node-conformance.md 1 file changed, 5 insertions(+), 4 deletions(-)
- [x] content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/configure-pod-container/quality-service-pod.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/tasks/debug-application-cluster/debug-pod-replication-controller.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/extend-kubectl/kubectl-plugins.md 1 file changed, 3 insertions(+), 3 deletions(-)
- [x] content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md 1 file changed, 27 insertions(+), 26 deletions(-)
- [x] content/en/docs/tasks/run-application/horizontal-pod-autoscale.md 1 file changed, 7 insertions(+), 1 deletion(-)
- [x] content/en/docs/tutorials/kubernetes-basics/_index.html 1 file changed, 1 insertion(+)
- [x] content/en/docs/tutorials/stateful-application/zookeeper.md 1 file changed, 4 insertions(+), 4 deletions(-)
- [x] content/en/training/_index.html 1 file changed, 22 insertions(+), 15 deletions(-)

2 files are deleted
- [x] content/en/docs/setup/production-environment/turnkey/_index.md
- [x] content/en/docs/tasks/administer-cluster/cluster-management.md

7 files are already updated in Korean documents
- [x] content/en/docs/concepts/scheduling-eviction/assign-pod-node.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/workloads/controllers/statefulset.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/contribute/new-content/overview.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/setup/release/notes.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/tools/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/examples/pods/pod-with-pod-affinity.yaml 1 file changed, 2 insertions(+), 2 deletions(-)

1 file is updated by #25287 ( @Arhell ) 
- [x] content/en/case-studies/newyorktimes/index.html 1 file changed, 3 insertions(+), 4 deletions(-)

### CODE REVIEW

**Comment by ianychoi** [comment]
I am just adding a part of my first round of review.. Thank you @pjhwa for such a great job

File: content/ko/case-studies/newyorktimes/index.html [hunk 1]
```diff
@@ -26,10 +26,9 @@ <h2>Impact</h2>
 
 <p>Speed of delivery increased. Some of the legacy VM-based deployments took 45 minutes; with Kubernetes, that time was "just a few seconds to a couple of minutes," says Engineering Manager Brian Balser. Adds Li: "Teams that used to deploy on weekly schedules or had to coordinate schedules with the infrastructure team now deploy their updates independently, and can do it daily when necessary." Adopting Cloud Native Computing Foundation technologies allows for a more unified approach to deployment across the engineering staff, and portability for the company.</p>
 
-{{< case-studies/quote author="Deep Kapadia, Executive Director, Engineering at The New York Times">}}
-<iframe style="padding:1%:" width="380" height="215" src="https://www.youtube.com/embed/DqS_IPw-c6o" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
-<iframe style="padding:1%:" width="380" height="215" src="https://www.youtube.com/embed/Tm4VfJtOHt8" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
-<br>
+{{< case-studies/quote author="Deep Kapadia, Executive Director, Engineering at The New York Times" >}}
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/case-studies/newyorktimes/index.html]: This PR doesn't have this file.

File: content/ko/docs/concepts/policy/resource-quotas.md [hunk 2]
```diff
@@ -74,8 +74,7 @@ API 서버 `--enable-admission-plugins=` 플래그의 인수 중 하나로
 | `limits.memory` | 터미널이 아닌 상태의 모든 파드에서 메모리 제한의 합은 이 값을 초과할 수 없음. |
 | `requests.cpu` | 터미널이 아닌 상태의 모든 파드에서 CPU 요청의 합은 이 값을 초과할 수 없음. |
 | `requests.memory` | 터미널이 아닌 상태의 모든 파드에서 메모리 요청의 합은 이 값을 초과할 수 없음. |
-| `hugepages-<size>` | 터미널 상태가 아닌 모든 파드에 걸쳐서, 지정된 사이즈의
-휴즈 페이지 요청은 이 값을 초과하지 못함. |
+| `hugepages-<size>` | 터미널 상태가 아닌 모든 파드에 걸쳐서, 지정된 사이즈의 휴즈 페이지 요청은 이 값을 초과하지 못함. |
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/concepts/policy/resource-quotas.md:77]: This PR updates just the change of line break in the original document.
As you said, it seems necessary to change the '휴즈 페이지' in the document to 'huge page' in other PR.

File: content/ko/docs/setup/best-practices/cluster-large.md [hunk 3]
```diff
@@ -1,122 +1,121 @@
...
+## 컨트롤 플레인 컴포넌트
 
-큰 쿠버네티스 클러스터를 설정할 때는 다음 이슈들을 고려해야 한다.
+대규모 클러스터의 경우, 충분한 컴퓨트 및 기타 리소스가 있는 컨트롤 플레인이
+필요하다.
 
-### 쿼터 문제
+일반적으로 장애 영역 당 하나 또는 두 개의 컨트롤 플레인 인스턴스를
+실행하고, 해당 인스턴스를 수직으로 먼저 확장한 다음 (수직) 규모로 하락하는
+지점에 도달한 후 수평으로 확장한다.
```
Comment (seokho-son) [inline content/ko/docs/setup/best-practices/cluster-large.md:42]:
```suggestion
일반적으로 장애 영역 당 하나 또는 두 개의 컨트롤 플레인 인스턴스를
실행하고, 해당 인스턴스를 수직으로(vertically) 먼저 확장한 다음 (수직) 규모로 하락하는
지점에 도달한 후 수평으로(horizontally) 확장한다.
```
오해의 소지를 줄이기 위해서 한영병기하면 좋을 것 같습니다.. ^^

File: content/ko/docs/setup/best-practices/cluster-large.md [hunk 4]
```diff
@@ -1,122 +1,121 @@
...
+실행하면 가용성이 낮아진다.
 
 ### etcd 저장소
 
-큰 클러스터의 성능 향상을 위해, 우리는 이벤트를 각각의 전용 etcd 인스턴스에 저장한다.
+큰 클러스터의 성능 향상을 위해, 사용자는 이벤트 오브젝트를 각각의
+전용 etcd 인스턴스에 저장한다.
 
 클러스터 생성시의 부가 스트립트이다.
+클러스터 생성 시에 다음을 수행할 수 있다(사용자 정의 도구 사용).
```
Comment (seokho-son) [inline content/ko/docs/setup/best-practices/cluster-large.md:61]:
```suggestion
클러스터 생성 시에 (사용자 도구를 사용하여) 다음을 수행할 수 있다.
```
custom tooling 은 아마 각자의 도구나 일반적인 방식을 의미하는 것 같습니다. 그래서 조금이나마 느낌을 살리고자 "정의" 는 제외해 보았습니다.

File: content/ko/docs/setup/best-practices/cluster-large.md [hunk 5]
```diff
@@ -1,122 +1,121 @@
...
+성능이 저하된다.
+
+클러스터 애드온 리소스 문제가 발생하지 않도록, 노드가 많은 클러스터를
+만들 때, 다음 사항을 고려한다.
+
+* 일부 애드온은 수직으로 확장된다. 클러스터 또는 전체 장애 영역을
+  제공하는 애드온 레플리카가 하나 있다. 이러한 애드온의 경우, 클러스터를 확장할 때
+  요청과 제한을 늘린다.
+* 많은 애드온은 수평으로 확장된다. 더 많은 파드를 실행하여 용량을 추가하지만,
+  매우 큰 클러스터에서는 CPU 또는 메모리 제한을 약간 높여야할 수도 있다.
```
Comment (seokho-son) [inline content/ko/docs/setup/best-practices/cluster-large.md:102]:
```suggestion
  매우 큰 클러스터에서는 CPU 또는 메모리 제한을 약간 높여야 할 수도 있다.
```

File: content/ko/docs/setup/best-practices/cluster-large.md [hunk 6]
```diff
@@ -1,122 +1,121 @@
...
+  VerticalPodAutoscaler는 _recommender_ 모드에서 실행되어 요청 및 제한에 대한
+  제안 수치를 제공할 수 있다.
+* 일부 애드온은 노드 당 하나의 복사본으로 실행되며 DaemonSet: 예를 들어 노드 수준 로그 집계 자이다. 수평 확장 애드온의 경우와 유사하게 CPU 또는 메모리 제한을 약간 높여야할 수도 있다.
+* 일부 애드온은 {{< glossary_tooltip text="데몬셋(DaemonSet)"
+  term_id="daemonset" >}}에 의해 제어되는 노드 당 하나의 복사본으로 실행된다(예: 노드 수준 로그 수집기). 수평
+  확장 애드온의 경우와 유사하게, CPU 또는 메모리 제한을 약간 높여야
+  할 수도 있다.
```
Comment (seokho-son) [inline content/ko/docs/setup/best-practices/cluster-large.md:109]:
```suggestion
* 일부 애드온은 {{< glossary_tooltip text="데몬셋(DaemonSet)"
  term_id="daemonset" >}}에 의해 제어되는 노드 당 하나의 복사본으로 실행된다(예: 노드 수준 로그 수집기). 수평
  확장 애드온의 경우와 유사하게, CPU 또는 메모리 제한을 약간 높여야
  할 수도 있다.
```

해당 라인은 삭제해야 할 것 같습니다. 혹시 모르니 확인 부탁드립니다.. ^^

원문
``` 
* Some addons run as one copy per node, controlled by a {{< glossary_tooltip text="DaemonSet"
  term_id="daemonset" >}}: for example, a node-level log aggregator. Similar to
  the case with horizontally-scaled addons, you may also need to raise CPU or memory
  limits slightly.
```

**Review by seokho-son — COMMENTED** [review]
@pjhwa 일단 제가 1차로 리뷰해보았습니다.. ^^

File: content/ko/docs/setup/best-practices/cluster-large.md (hunk 3 above)
Comment (pjhwa) [reply content/ko/docs/setup/best-practices/cluster-large.md:42]: 반영하겠습니다.

File: content/ko/docs/setup/best-practices/cluster-large.md (hunk 4 above)
Comment (pjhwa) [reply content/ko/docs/setup/best-practices/cluster-large.md:61]: 반영하겠습니다.

File: content/ko/docs/setup/best-practices/cluster-large.md (hunk 5 above)
Comment (pjhwa) [reply content/ko/docs/setup/best-practices/cluster-large.md:102]: 반영하겠습니다.

File: content/ko/docs/setup/best-practices/cluster-large.md (hunk 6 above)
Comment (pjhwa) [reply content/ko/docs/setup/best-practices/cluster-large.md:109]: 네, 맞습니다. 지운다는게 깜빡했네요. ^^;

**Comment by pjhwa** [comment]
@seokho-son 리뷰 감사합니다. 
의견주신 내용은 모두 반영하였습니다.

**Comment by seokho-son** [comment]
@ianychoi 브랜치 관리 차원에서 해당 PR은 최대한 빨리 처리하고자 합니다..!
리뷰 중이신 내용이나 추가 의견이 있으시면 알려주세요 ~~ ^^ 특이사항이 없으면 제가 최종 검토하여 승인하도록 하겠습니다.
감사합니다 ~~ !

**Comment by pjhwa** [comment]
> @ianychoi 브랜치 관리 차원에서 해당 PR은 최대한 빨리 처리하고자 합니다..!
> 리뷰 중이신 내용이나 추가 의견이 있으시면 알려주세요 ~~ ^^ 특이사항이 없으면 제가 최종 검토하여 승인하도록 하겠습니다.
> 감사합니다 ~~ !

@seokho-son 스쿼시 작업을 하려니 뭔가 로컬에서 꼬였는지 다른 커밋이 섞여들어가네요. 쩝... (일단 그래서 스쿼시 작업은 못하고 있습니다. ㅠㅠ)

**Comment by seokho-son** [comment]
@pjhwa 해당 PR은 오늘 저녁까지 특별히 추가 의견이 없는 경우 머지하도록 하겠습니다 ~ (1.20 릴리스 일정 등을 고려하여)
스쿼시 관련해서는 편하신 방법으로 처리하면 좋을 것 같습니다. (PR 다시 열기, 스쿼싱 문제 해결 등등)

**Comment by pjhwa** [comment]
> @pjhwa 해당 PR은 오늘 저녁까지 특별히 추가 의견이 없는 경우 머지하도록 하겠습니다 ~ (1.20 릴리스 일정 등을 고려하여)
> 스쿼시 관련해서는 편하신 방법으로 처리하면 좋을 것 같습니다. (PR 다시 열기, 스쿼싱 문제 해결 등등)

@seokho-son 오늘 저녁 무렵까지 추가 의견이 없다면, 새로운 PR 열겠습니다.

**Pull request closed without merging**